tests/integration_tests/int_test_git_magic.py

```diff
@@ -0,0 +1,179 @@
+import logging
+import os
+import shutil
+import sys
+import tempfile
+import unittest
+from unittest.mock import Mock, patch
+
+from git import Repo
+
+from colony import branch_utils, shell
+from colony.constants import UNCOMMITTED_BRANCH_NAME
+from tests.helpers.repo_utils import (
+    achieve_dirty_and_untracked_repo,
+    add_untracked,
+    create_clean_repo,
+    make_repo_dirty,
+    readonly_handler,
+)
+
+logging.getLogger("git").setLevel(logging.WARNING)
+logger = logging.getLogger(__name__)
+
+
+class GitMagicTests(unittest.TestCase):
+    def setUp(self, *mocks) -> None:
+        # self._patcher()
```

Comment:
remove this comment?

Reply:
seems a ghost has added that...
will remove it